### DISCUSSION TITLE
Please tell me your love animals!

### DISCUSSION THREAD

**Discussion by mmikuu** [NAIST-SE/AttractingContributors#1109] [2024 Attraction Contributions]
好きな動物をおしえてください！
Please draw your love animal in README.md and submit it to my project.
https://github.com/mmikuu/AttractingContributors-animal

**Comment by Mont9165** [comment]
🐈‍⬛ 🐈‍⬛ 🐈‍⬛ 🐈‍⬛ 🐈‍⬛
